GymBro/Views/RoutineViews/AddExerciseButton.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AddExerciseButton.swift
+//  GymBro
+//
+//  Created by Claire on 4/11/23.
+//
+//  This view creates a pop-up sheet including a wheel picker for user to
+//  choose which exercise to add to the routine list of day.
+//  By clicking "New" button, the user can add an new exercise that does not
+//  exist to the wheel picker.
+
+import SwiftUI
+
+struct AddExerciseButton: View {
```

Comment:
Documentation guideline:

If it's possible, please add some simple documentation. 
Check the "🎮Xcode Document" section in the GuideLines.md files.